builder/image/Dockerfile

```diff
@@ -13,6 +13,10 @@ RUN curl -sSL -o /usr/local/bin/etcdctl https://s3-us-west-2.amazonaws.com/opdem
 RUN curl -sSL -o /usr/local/bin/confd https://s3-us-west-2.amazonaws.com/opdemand/confd-v0.5.0-json \
     && chmod +x /usr/local/bin/confd
 
+# install herokuish
+RUN curl -sSL https://github.com/aledbf/herokuish/releases/download/v0.1.2/herokuish_0.1.2_linux_x86_64.tgz \
```

Comment:
I think it would be better to use the official release and upstream all changes to have full BP test coverage. @progrium would you mind creating a release when all changes required for this PR are in herokuish?